INaturalistIOS/ObservationsViewController.m

```diff
@@ -638,6 +632,13 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     return [sectionInfo numberOfObjects];
 }
 
+//- (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath{
```

Comment:
do you have plans for this method? if not, please no commented out code, thanks!